ports/windows/.appveyor.yml

```diff
@@ -1,6 +1,6 @@
 environment:
   # Python version used
-  MICROPY_CPYTHON3: c:/python34/python.exe
+  MICROPY_CPYTHON3: c:/python35/python.exe
```

Comment:
Until we fully switch to Python 3.5 as the reference, it'd be better to provide a `.py.exp` file for the test instead of this change.